compositor_render/src/renderer.rs

```diff
@@ -248,6 +248,12 @@ impl WgpuCtx {
             None,
         ))?;
 
+        let shader_header =
+            naga::front::wgsl::parse_str(include_str!("transformations/shader_header.wgsl"))
+                .expect(
+                    "couldn't find shader header file, which is necessary for shader validation",
```

Comment:
include_str reads the file at compile time, so this error can only happen if the file exists but it's not valid. If that is the case, this error is a bit misleading.